rpm/BUILD

```diff
@@ -23,6 +23,10 @@ bzl_library(
     name = "lib",
     srcs = [
         "rules.bzl",
+        "@rules_pkg//:rpm.bzl",
+    ],
+    deps = [
+        "@rules_pkg//doc_build:rules_pkg_lib",
```

Comment:
Pull in dependencies required for stardoc's `bzl_library` to run
In this case, `rules_pkg` actually already provides a `doc` package with a `bzl_library` we can depend on